editor/src/components/canvas/canvas-strategies/absolute-move-strategy.ts

```diff
@@ -1,9 +1,18 @@
 import { MetadataUtils } from '../../../core/model/element-metadata-utils'
+import { toString } from '../../../core/shared/element-path'
```

Comment:
Looks like these imports were accidentally committed to the wrong branch?

Reply:
oooops

Reply:
fixed